packages/dev-utils/src/generate-documentation-table.ts

```diff
@@ -20,7 +20,7 @@ const options: OptionDocumentation[] = [
     name: "authToken",
     type: "string",
     fullDescription:
-      "The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/settings/account/api/auth-tokens/. Required scopes: project:releases (and org:read if setCommits option is used).\n\nThis value can also be specified via the `SENTRY_AUTH_TOKEN` environment variable.",
+      "The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/. Required scopes: project:releases (and org:read if setCommits option is used).\n\nThis value can also be specified via the `SENTRY_AUTH_TOKEN` environment variable.",
```

Comment:
```suggestion
      "The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/.\n\nThis value can also be specified via the `SENTRY_AUTH_TOKEN` environment variable.",
```

no scopes to set manually there!